packages/shopify-codemod/test/transforms/mocha-context-to-global-references.test.js

```diff
@@ -0,0 +1,28 @@
+import 'test-helper';
+import mochaContextToGlobalReferences from 'mocha-context-to-global-references';
+
+describe('mochaContextToGlobalReferences', () => {
+  it('renames the member expression to use a test global', () => {
+    expect(mochaContextToGlobalReferences).to.transform('mocha-context-to-global-references/rename', {
+      testContextToGlobals: {
+        sinon: {properties: ['spy', 'stub', 'server']},
+      },
+    });
+  });
+
+  it('replaces a contextual reference to a global reference', () => {
+    expect(mochaContextToGlobalReferences).to.transform('mocha-context-to-global-references/replace', {
+      testContextToGlobals: {
+        testClock: {
```

Comment:
If I'm reading correctly, it means, "take this new name, and use it to replace any of these existing property names".  I have to reverse the intent of the transform, and that seems counter-intuitive.

I feel it'd be clearer to have, "take this name, and replace it with this one".  e.g.,
```
{
  testContextToGlobals: [
    { from: 'clock', to: 'testClock', replace: true },
    { from: 'shouldBeReplaced', to: 'testGlobal' },
    { from: 'mustBeReplaced', to: 'testGlobal' }
  ]
}
```
... or some format where old=>new is the mapping.